sklearn/preprocessing/_data.py

```diff
@@ -1951,24 +1951,45 @@ def _more_tags(self):
 
 
 class KernelCenterer(TransformerMixin, BaseEstimator):
-    """Center a kernel matrix.
+    r"""Center an arbitrary kernel matrix :math:`K`.
 
-    Let K(x, z) be a kernel defined by phi(x)^T phi(z), where phi is a
-    function mapping x to a Hilbert space. KernelCenterer centers (i.e.,
-    normalize to have zero mean) the data without explicitly computing phi(x).
-    It is equivalent to centering phi(x) with
-    sklearn.preprocessing.StandardScaler(with_std=False).
+    Let define a kernel :math:`K` such that:
+
+    .. math::
+        K(X, Y) = \phi(X) . \phi(Y)^{T} \ ,
```

Comment:
I don't understand why you need the final `\  ,`.

Reply:
I now get it's because the sentence continues after with "where :math:`\phi(X)` is a function mapping..." Maybe you can just remove the coma and start a new sentence with `:math:`\phi(X)` is a function mapping ...` on the next line.

Reply:
as you wish.